inst/examples/dashboard/about.R

```diff
@@ -0,0 +1,179 @@
+shiny_appsilon_card <- div(
+  class = "card ms-depth-8",
+  Stack(
+    tokens = list(childrenGap = 10),
+    a(
+      href = "https://appsilon.com/",
+      span(class = "ms-font-su ms-fontColor-themePrimary", "Appsilon")
+    ),
+    Stack(
+      wrap = FALSE,
+      horizontal = TRUE,
+      horizontalAlign = TRUE,
+      verticalFill = TRUE,
+      tokens = list(childrenGap = 12),
+      a(
+        href = "https://appsilon.com/",
+        Image(
+          src = "appsilon-logo.png",
+          width = "120px",
+          height = "70px"
+        )
+      ),
+      Stack(
+        tokens = list(childrenGap = 5),
+        Text(
+          variant = "large", "This Project is Developed By Appsilon",
+          block = TRUE
+        ),
+        Text(
+          "We create, maintain, and develop Shiny applications for
+          enterprise customers all over the world. Appsilon provides
+          scalability, security, and modern UI/UX with custom R
+          packages that native Shiny apps do not provide. Our team is
+          among the world’s foremost experts in R Shiny and has made
+          a variety of Shiny innovations over the years. Appsilon is
+          a proud RStudio (Posit) Full Service Certified Partner."
+        )
+      )
+    )
+  )
+)
+
+shiny_fluent_card <- div(
+  class = "card ms-depth-8",
+  Stack(
+    tokens = list(childrenGap = 10),
+    a(
+      href = "https://appsilon.github.io/shiny.fluent/index.html",
+      span(class = "ms-font-su ms-fontColor-themePrimary", "shiny.fluent")
+    ),
+    Stack(
+      wrap = FALSE,
+      horizontal = TRUE,
+      horizontalAlign = TRUE,
+      verticalFill = TRUE,
+      tokens = list(childrenGap = 12),
+      a(
+        href = "https://appsilon.github.io/shiny.fluent/index.html",
+        Image(
+          src = "img/shiny-fluent.png",
+          width = "120px",
+          height = "120px"
+        )
+      ),
+      Stack(
+        tokens = list(childrenGap = 5),
+        Text(
+          variant = "large", "Check the rest of the App to Learn More...",
+          block = TRUE
+        ),
+        Text(
+          "We believe that a great UI plays a huge role in the success of
+          application projects. shiny.fluent gives your apps: - beautiful,
+          professional look - rich set of components easily usable in Shiny -
+          fast speed of development that Shiny is famous for. As Fluent UI is
+          built in React, shiny.fluent is based on another package called
+          shiny.react, which allows for using React libraries in Shiny."
```

Comment:
A list needs reformatting in this text...
```suggestion
          "We believe that a great UI plays a huge role in the success of
          application projects. shiny.fluent gives your apps: beautiful,
          professional look, rich set of components easily usable in Shiny
          and fast speed of development that Shiny is famous for. As Fluent UI is
          built in React, shiny.fluent is based on another package called
          shiny.react, which allows for using React libraries in Shiny."
```

Reply:
Changed the text